### PR TITLE
DOCSP-44007-update-ai-and-data-usage

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Updating AI and Data Usage page (wording already approved by legal)

## STAGING

https://deploy-preview-98--docs-mongodb-vscode.netlify.app/ai-data-usage/

## JIRA

https://jira.mongodb.org/browse/DOCSP-44007

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jason-price-mongodb — APPROVED** [review]
@ltran-mdb2 LGTM. Cheers.

File: source/ai-data-usage.txt [hunk 1]
```diff
@@ -23,14 +23,22 @@ The |copilot| is powered by `GitHub Copilot
...
-connection strings, credentials, or rows/documents from your databases. 
+GitHub Copilot stores the following information:
+
+- The full text of your natural language prompt
+- The full text of the Copilot generated response
+- User engagement and feedback data 
+
+The information that is sent will not be shared with any other third
+parties. We do not send out database connection strings, credentials, or
+rows/documents from your databases.
```
Comment (Anemy) [inline source/ai-data-usage.txt:41]: If the user enabled the setting for `Enable sending sample field values with the VSCode copilot chat @MongoDB participant /query command` then documents could make their way into the prompts. Should we call that out?

Comment (ltran-mdb2) [reply source/ai-data-usage.txt:41]: I added a mention of this setting!

**Comment by ltran-mdb2** [comment]
Thanks for your feedback @Anemy ! Could you take another look when you get a chance?

**Review by Anemy — APPROVED** [review]

**Pull request merged**